wagtail/models/__init__.py

```diff
@@ -2659,8 +2659,14 @@ class Meta:
 
 
 class RevisionQuerySet(models.QuerySet):
+    def page_revisions_q(self):
+        return Q(base_content_type=get_default_page_content_type())
+
     def page_revisions(self):
-        return self.filter(base_content_type=get_default_page_content_type())
+        return self.filter(self.page_revisions_q())
+
+    def not_page_revisions(self):
+        return self.exclude(self.page_revisions_q())
```

Comment:
Following the pattern in https://github.com/wagtail/wagtail/blob/main/wagtail/query.py